draft-ietf-cdni-uri-signing.xml

```diff
@@ -544,6 +544,33 @@
             claim if no Client IP claim existed in the received
             signed JWT.</t>
         </section>
+        <section anchor="cdnib_claim" title="CDNI Base (cdnib) claim">
+            <t>Base (cdnir) [optional] -
+            The base path for which the token is intended. Its type is a
+            JSON string. It contains a <xref target="RFC3986">URI</xref>,
+            except that any or all of the scheme, host, or port MAY be
+            omitted. This URI MUST NOT contain User Information
+            (<xref target="RFC3986" /> Section 3.2.1), Query (Section 3.4),
+            or Fragment (Section 3.5) components.</t>
+
+            <t>The scheme is terminated by a colon (':'), which MUST be
+            omitted if the scheme is. The authority is preceeded by a
+            double slash ("//"), which MUST be omitted if the authority
+            is, and terminated by the next slash ("/"), which begins the
+            required path, or the end of the URI.</t>
+
+            <t>The base claim MUST NOT be used as a sole basis to either
+            accept or reject a request. Rather, other claims use the
+            information claimed here.</t>
```

Comment:
I agree here, "claimed" is not great here, since Claim has a very specific meaning in this draft. "asserted herein" or "provided here" is better.

Reply:
Yeah. You're getting to what I dislike the most about this particular "claim": it does not by itself restrict anything. It's an informational "claim", used by other claims to carry data. Specifically, the cdniuc of `b:` "references" it and turns it into an actual restriction. And processing of the cdnistt claim requires the host and the path.